.azure-pipelines/azure-pipelines-merge.yml

```diff
@@ -73,7 +73,7 @@ jobs:
       Python36:
         python.version: '3.6.8'
```

Comment:
Can this be 3.6.x ? and will 3.x pick the latest one available?

Reply:
3.6.8 is the latest in 3.6 series.. so we can leave this as it is